Sources/SQLiteStore.swift

```diff
@@ -74,12 +74,13 @@ public final class SQLiteStore: LocalStorage {
      
      - Warning: The default SQLite file location for the `LegacySQLiteStore` and `SQLiteStore` are different. If the app was depending on CoreStore's default directories prior to 2.0.0, make sure to use the `SQLiteStore.legacy(...)` factory methods to create the `SQLiteStore` instead of using initializers directly.
      */
-    public init() {
+    public init(secure: Bool = false) {
```

Comment:
I'd rather implement this as a separate store, or a subclass of `SQLiteStore`. The use of `NSFileProtectionType`s have some nuanced effects, and the errors that may occur need to be expressed in `CoreStoreError`.

Reply:
Can be good, I had tried a quick approach without too many changes, prob a subclass is even better